collector: Internal telemetry

# Problem Description
This snippet doesn't match the allowed values in the table above it.

Furthermore, starting a collector using this results in the following (depending on collector version).

## Collector v0.23.0
```
panic: reflect: Key of non-map type interface {}
```

## Collector v0.12.0

`'service.telemetry.logs' has invalid keys: processors`

```
service:
  telemetry:
    logs:
      processors:
        - batch:
            exporter:
              otlp:
                protocol: http/protobuf
                endpoint: https://backend:4317
```

## Sidenote: Port

`endpoint: https://backend:4317` this should be `:4318` no?

--- a/content/en/docs/collector/internal-telemetry.md
+++ b/content/en/docs/collector/internal-telemetry.md
@@ -144,7 +144,7 @@ service:
             exporter:
               otlp:
                 protocol: http/protobuf
-                endpoint: https://backend:4317
+                endpoint: https://backend:4318
 ```
 
 ### Configure internal traces

